### ISSUE TITLE
Mirakel compatibility ?

### ISSUE THREAD

**Issue by Dark-Fenix** [bitfireAT/davdroid#527]
Hi,

Looking for a task-manager for Android which can sync with ownCloud (i'm using the "Tasks" app, which works with Caldav), I found Mirakel very user-friendly. However, to enable the sync feature with a Caldav server, they explain that their specific version of Davdroid (avalaible through an apk on the website) is required until the official Davdroid app support Mirakel.

Is the compatibilty problem real ? Is there a planned update to fix it, or is there any workaround to make the 3-parts ownCloud-Davdroid-Mirakel working ?

Thanks in advance :)

**Comment by untitaker** [comment]
The compat problem is real.

However, as per #47 DavDroid now supports task sync via the dmfs Tasks app, it's up to the Mirakel devs to implement the same interface.

**Comment by Pastafarianist** [comment]
Mirakel developers seem to be losing interest in development (I've recently posted a bug in their bug tracker and was told that due to the lack of time they cannot rebuild DAVDroid-Mirakel, which would have solved my bug). It's probably better to stick to Tasks.

**Comment by devvv4ever** [comment]
We're trying to support Mirakel 3 which is currently in beta and will be released soon. If everything goes as expected DAVdroid will be able to sync with this and future versions of Mirakel!

**Comment by Dark-Fenix** [comment]
Thanks for the advice @Pastafarianist, but "Tasks" is still in beta : is it really reliable ? Is the sync with ownCloud ok ? I'll try but keep an eye on Mirakel, I love the widget :'(
@devvv4ever is there any release date for the next Mirakel ?

**Comment by untitaker** [comment]
Personally I've found Tasks to be much more reliable than Mirakel -- don't trust the labels. I also suppose that Task's data model is more compatible with the VTODO stuff in CalDAV while Mirakel seems to be built around Taskwarrior's data model, I suppose?

**Comment by bugzy** [comment]
@devvv4ever 
Mirakel 3.0 is now released. Just wondering whether the current DAVdroid will sync with Mirakel 3.0 or whether the DAVdroid devs will need to turn on compatibility in a future DAVdroid release.

**Comment by devvv4ever** [comment]
Yes it is planned to support Mirakel in one of the next minor releases of DAVdroid. We're pretty busy right now with our business but it will hopefully come in the near future.

**Comment by rfc2822** [comment]
Just replacing the authority and permissions doesn't seem to do the trick…
